doc/source/ray-air/examples/gptj_deepspeed_fine_tuning.ipynb

```diff
@@ -426,13 +426,13 @@
     "\n",
     "#### Training speed\n",
     "\n",
-    "As we are using data parallelism, each worker will operate on its own shard of the data. The batch size we set in `TrainingArgument` is the **per device batch size** (or, in other words, per worker batch size). By changing the number of workers, we can change the **effective batch size** and thus the time needed for training to complete. The effective batch size is then calculated as `per device batch size * number of workers * number of gradient accumulation steps`. As we add more workers, our effective batch size will rise and thus we will need less time to complete a full epoch. While the speedup is not exactly linear due to extra communication overheads, in many cases it can be close to linear.\n",
+    "As we are using data parallelism, each worker will operate on its own shard of the data. The batch size we set in `TrainingArguments` is the **per device batch size** (or, in other words, per worker batch size). By changing the number of workers, we can change the **effective batch size** and thus the time needed for training to complete. The effective batch size is then calculated as `per device batch size * number of workers * number of gradient accumulation steps`. As we add more workers, our effective batch size will rise and thus we will need less time to complete a full epoch. While the speedup is not exactly linear due to extra communication overheads, in many cases it can be close to linear.\n",
```

Comment:
```suggestion
    "As we are using data parallelism, each worker operates on its own shard of the data. The batch size set in `TrainingArguments` is the **per device batch size** (per worker batch size). By changing the number of workers, we can change the **effective batch size** and thus the time needed for training to complete. The effective batch size is then calculated as `per device batch size * number of workers * number of gradient accumulation steps`. As we add more workers, the effective batch size rises and thus we need less time to complete a full epoch. While the speedup is not exactly linear due to extra communication overheads, in many cases it can be close to linear.\n",
```